src/analysis/locate.mli

```diff
@@ -33,6 +33,7 @@ val from_string
   -> env:Env.t
   -> local_defs:Mtyper.typedtree
   -> pos:Lexing.position
+  -> [ `Any | `Type ]
```

Comment:
I think it doesn't hurt to make this more precise, so I'd suggest having:
```ocaml
    -> ?namespace:[ `Type | `Module | `Value | ... ]
```

Instead of this.

And at that point, exposing a proper (named) type might also make sense?